changelog/14474.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+auth/approle: add 'ttl' and 'num_uses' fields when generating SecretIDs
```

Comment:
would you please be more elaborative here? Like to indicate that the two fields are per secretID, yet if they are not configured, the role specific values are used.